### DISCUSSION TITLE
안녕하세요 번역작업관련해 컨트리뷰터로 참여하고싶습니다.

### DISCUSSION THREAD

**Discussion by cloakcat** [rust-kr/doc.rust-kr.org#12] [Q&A]
안녕하세요 영어원문과 비교해보니 챕터 2번부터 약간 내용이 업데이트된 부분들이 간간히 있어 
함께 번역작업하고자 요청드립니다.
참여 조건이 있다면 안내부탁드립니다.

**Comment by rinthel** [comment]
환영합니다! 관심 감사합니다~
별다른 참여 조건같은 것은 없습니다.
현재 이 저장소에서 작업하고 있는 2018년 버전 빌드 (https://rust-kr.github.io/doc.rust-kr.org/)과
2nd 버전 빌드 (https://rinthel.github.io/rust-lang-book-ko/)은 차이가 있을 수 있습니다.
2018년 버전 빌드는 /src 디렉토리에 있는 md 파일을,
그리고 2nd 버전 빌드는 /second-edition/src 디렉토리에 있는 md 파일을 빌드한 결과입니다.
readme에 적어놓은 내용을 참고하셔서 fork / 수정 후 PR 주시면 리뷰 후 머지하도록 하겠습니다-
